### DISCUSSION TITLE
Sharepoints on two different domains?

### DISCUSSION THREAD

**Discussion by Mattastics** [Barbarur/NovaPoint#38] [Q&A]
Our companies first sharepoint sight was created a few years ago with an old company email address. We still have a handful of users that use that dor documents and data.
But a year or so ago we got new email addresses on a new domain and created a new sharepoint site. I was hoping to use this to pull all of the documents off of the old one and onto the new but it looks like I may need a client id and tenant id for each domain. Anyone have any ideas on that?

**Comment by Barbarur** [comment]
Hi @Mattastics ,
Do you have 2 different tenants or 2 different sites on the same tenant?

You can have 2 or more email domains on the same tenant. It seems strange estoy have a full new tenant only due domain change.

If you have 2 sites on the same tenant you can use NovaPoint for cross-site file copy. Only a single Tenant ID and Client ID is needed.

But if you really have 2 tenants, NovaPoint doesn't support cross tenant migration yet.

**Reply by Mattastics** [reply]
It seems like we have two different tenants but honestly I'm not entirely sure how to tell but I will look into it. I have access to an old server we are about to decommision. Assuming they are on different tenats, I guess the best way to move all the files would be to download them all to that server then upload them all to the new sharepoint?

**Reply by Barbarur** [reply]
Hi @Mattastics 
Depending on the source and destination type of tenant you have few options for migrating the data:
- From SharePoint on-premises to SharePoint Online; You can use the SharePoint Migration tool (https://learn.microsoft.com/en-us/sharepointmigration/introducing-the-sharepoint-migration-tool) which is free or any other 3rd party tool, though they can be pricy.
- From SharePoint Online to SharePoint Online; Only 3rd party app cover this activity. My intention is to bring NovaPoint to be able to cover this, but I'm not there yet on the development.
- From SharePoint on-premises to SharePoint on-premises; I guess this is not your scenarion, but to add it to the list. This action is more about upgrade than migration: https://learn.microsoft.com/en-us/answers/questions/495085/how-to-migrate-data-from-sharepoint-2016-on-prem-t

**Reply by Mattastics** [reply]
I've only been at this position for about a month and had never dealt with shrepoint much before so I'm not entirely sure what our sitation is, but I am fairily confident our old one was on-prem but the new one was not. I had to coordinate with the actual owners on a few different settings with autentication but I was able to get it all done. Happy to share some things with you if it will help.
